service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java

```diff
@@ -77,6 +77,10 @@ public boolean isEldestChildLastNameEnabled() {
         return ldClient.boolVariation("eldest-child-last-name", createLDUser(), false);
     }
 
+    public boolean isSendOrderToOthersEnabled() {
```

Comment:
do we need toggle? The whole event is toggled off.

Reply:
discussed to user order.IsNewVersion instead